index.html

```diff
@@ -2,6 +2,7 @@
 <head>
   <title>GitHub Issues</title>
     <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
+    <script src="js/secrets.js"></script>
```

Comment:
Can you send me a pm on slack of a paraphrase of what's in secrets.js?

I'm just really curious how exactly you did this.  :)  Nice work.